tests/language-feature/capability/var-capability-incompatible.slang

```diff
@@ -0,0 +1,22 @@
+//TEST:SIMPLE(filecheck=CHECK1):-target spirv
+//TEST:SIMPLE(filecheck=CHECK2):-target spirv
```

Comment:
there should be no need to run compile twice. Just a single `//TEST:SIMPLE(filecheck=CHECK)` line should suffice.